index.html

```diff
@@ -4268,50 +4267,53 @@ <h3>Language Declarations</h3>
-content-sniffing when attempting to determine the language.
-        </li>
       </ul>
     </section>
 
     <section class="informative">
-      <h3>Text Layout</h3>
+      <h3>Text direction</h3>
 
   <p>
-When expressing human-readable text content in the data model, it is often
-useful to encode the text direction for languages. In some cases, the layout
-of text is complicated and multi-dimensional (for example, in the case of
-<a href="https://en.wikipedia.org/wiki/Ruby_character">Ruby characters</a>).
-The expressibility of text direction and layout is often a function of the
-syntax utilized. Options for encoding text layout information in are described
-below:
-      </p>
-
-      <ul>
-        <li>
+When expressing human-readable text content in the data model, it is important
+to be able to explicitly encode the text direction. JSON and its derived formats
+use the UTF-8 encoding of [[Unicode]] for serializing and transmitting text.
+This enables text direction to be directly determined in simple cases,
+but the mechanism is insufficient for general use.
+  </p>  
+    
+  <p>
 JSON-LD supports the expression of text layout information via the
 <code>rdf:HTML</code> string literal type, which is described in JSON-LD 1.0,
 <a href="https://www.w3.org/TR/json-ld11/#typed-values">Section 4.2.1: Typed Values</a>.
 This mechanism enables a developer to use the internationalization features
 present in HTML to express language information. For example, consider the
 following key-value pair:
-<code>"nameHtml": "&lt;span dir="rtl" lang="ar">أمل&lt;/span>"</code>.
-If the JSON-LD Context expressed that any value associated with the <code>nameHtml</code>
-property as being of type <code>rdf:HTML</code>, then software agents
-rendering the property would be able to deterministically identify the
-language in use as well as the text direction of the language.
-        </li>
-        <li>
-Both JSON and JSON-LD support raw
-<a href="https://en.wikipedia.org/wiki/Bi-directional_text">UTF-8 Bi-Directional</a>
-text when used to express text direction. This mechanism is limited and
-often does not support more complex text layout mechanism provided by the
-mechanism expressed above.
-        </li>
-      </ul>
+<code>"nameHtml": "&lt;span dir="rtl" lang="ar">HTML و CSS: تصميم و إنشاء مواقع الويب&lt;/span>"</code>.
+If the JSON-LD Context states that any value associated with the <code>nameHtml</code>
+property is of type <code>rdf:HTML</code>, then software agents
+have sufficient information to deterministically identify the text direction of the language.
+  </p>
+      
+    </section>
+    
+    <section class="informative">
+      <h3>Ruby Text</h3>
+  <p>
+In some situations it is common to use
+<a href="https://en.wikipedia.org/wiki/Ruby_character">Ruby characters</a>)
+to ensure that text information is clear and as useful as possible.
+JSON-LD's support for HTML data allows text to be encoded with ruby, supporting
+these cases. 
```

Comment:
Actually, my understanding is that what would be represented by ruby in running text is often represented in databases of string (such as people's names, addresses, etc.) by a separate pronunciation field within a record, rather than by using ruby markup.  I don't have any hard evidence of this or the extent to which ruby is or isn't used, but i did hear it from a Japanese person recently in the context of web publishing metadata.

Reply:
Yes, I know of other examples where this is done - and I don't know how widespread it is either although I presume it is much more common that using HTML as the format to manage that kind of information.
I'll think about it, and see what the group and editor's think, but as I believe the change would be non-substantive, I will look at it in a week or so...